### DISCUSSION TITLE
How to make Netbox read-only for maintenance?

### DISCUSSION THREAD

**Discussion by llamafilm** [netbox-community/netbox#15231] [Q&A]
How is maintenance mode supposed to work? I tried enabling Maintenance Mode in the config revision and then restarted netbox service.  Now it shows a banner at the top of the screen but I can still modify devices.  Maintenance mode doesn't seem to have any effect. I thought it should prevent any changes in this mode… I'm running version 3.7.0.

**Comment by candlerb** [comment]
AFAIK, you need to stick the database into read-only mode yourself. Maintenance mode just prevents Netbox from trying to update the last login time (which would cause logins to fail)
